restaurants.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Best Restaurants</title>
+    <link rel="stylesheet" href="css/restaurants.css">
```

Comment:
<!-- I noticed here the CSS path folder should be "CSS/restaurants.css," it seems like this small error is not allowing the css to render to your restaurant and things to do page. -->